lib/tls.js

```diff
@@ -44,7 +44,7 @@ const { getRootCertificates, getSSLCiphers } = internalBinding('crypto');
 const { Buffer } = require('buffer');
 const EventEmitter = require('events');
 const { URL } = require('internal/url');
-const DuplexPair = require('internal/streams/duplexpair');
+const { DuplexPair } = require('stream');
```

Comment:
Should this refer to _stream_duplexpair instead?

Reply:
All the underscore-prefixed modules are considered legacy, and we definitely want to avoid introducing new ones or their usage